src/passwordPenalty.js

```diff
@@ -12,16 +12,11 @@ export default function penaltyPoints(password = "") {
   if(password === null) return 0;
   if (typeof password !== "string") password = String(password);
 
-  // * * * * * * * * * * * * * * * * * * * * * * * * * * * *
-  // * * * INSERT YOUR CODE HERE * * * * * * * * * * * * * *
-    //
-    let regex = (/([a-z0-9])\1+/gi)
-    let count = 0;
+    // Here this line of code below groups similar characters together
+    const regex = (/([a-z0-9])\1+/gi)
    let matchArr = (password.match(regex) || 0)
      if(matchArr === 0) return 0;
```

Comment:
Is this if needed? There will no way this matchArr will be 0 because the math method return the array of matching values or null if there are no matching values. 

So there is no role this if playing, you can check.

Reply:
The zero is the default value incase no match is found. If I try removing it, It causes errors and a test case fails

Reply:
..